index.html

```diff
@@ -638,218 +665,87 @@ <h2>
           When <dfn>clearWatch()</dfn> is invoked, the user agent MUST:
         </p>
         <ol class="algorithm">
-          <li>If [=this=]'s {{Geolocation/[[watchTasks]]}}[|watchId|]
-          [=map/exists=], [=map/remove=] [=this=]'s
-          {{Geolocation/[[watchTasks]]}}[|watchId|].
+          <li>[=List/Remove=] |watchId| from [=this=]'s
+          {{Geolocation/[[watchIDs]]}}.
           </li>
         </ol>
       </section>
       <section>
         <h2>
-          Request position
+          Request a position
         </h2>
         <p>
-          <dfn>Request position</dfn> by passing a {{PositionCallback}}
-          |successCallback:PositionCallback|, a {{PositionErrorCallback}}`?`
-          |errorCallback:PositionErrorCallback|, {{PositionOptions}}
-          |options:PositionOptions|, a {{boolean}} |repeats:boolean|, and
-          optionally (and only if |repeats| is true) a |previousId:long|.
+          To <dfn>request a position</dfn>, pass a {{PositionCallback}}
+          |successCallback:PositionCallback|, a {{PositionErrorCallback?}}
+          |errorCallback:PositionErrorCallback?|, {{PositionOptions}}
```

Comment:
The type declarations in the method steps for `getCurrentPosition()` and `watchPosition()` should also be updated to use "PermissionErrorCallback?".

Reply:
Indeed! Fixed.